Content.Server/Administration/Logs/AdminLogManager.cs

```diff
@@ -288,30 +296,26 @@ private void Add(LogType type, LogImpact impact, string message, JsonDocument js
             return;
         }
 
+        // TODO fix AdminLogEntity
+        // Or maybe just remove it altogether.
+        foreach (var entity in entities)
+        {
+            entity.Uid = 0;
+        }
```

Comment:
I don't fully understand why this is needed, but the database seems to explode if the entity uids aren't all set to 0, which is old behaviour anyways.

There was also an old comment that just said `// TODO set the id too whenever we feel like running a migration for 10 hours` which might be related?

My vague understanding is that a piece of data has to be uniquely identifiable using all the fields with the [Key] attribute, meaning that `AdminLogEntity` would need to get a `LogId` & `RoundId` like in `AdminLogPlayer`?
But if thats the case why doesn't it currently explode when all uids are 0???